### PR TITLE
[DOC] add examples for select*

### PR DESCRIPTION
**Pull request by samukweku**
- Add examples for the select functions

- @ericmjl

### CODE REVIEW

**Comment by ericmjl** [comment]
🚀 Deployed on https://deploy-preview-1244--pyjanitor.netlify.app

File: janitor/functions/select.py [hunk 1]
```diff
@@ -27,21 +27,180 @@ def select_columns(
...
+        num_wings       0     2
+
+        Selection with a callable on a level:
+        >>> df.select_columns({"animal":lambda f: f.str.endswith('t')})
+        class      mammal
+        animal        cat   bat
+        locomotion  walks flies
+        num_legs        4     2
+        num_wings       0     2
+
```
Comment (thatlittleboy) [inline janitor/functions/select.py:203]: nit: Just one empty newline between consecutive sections will suffice.

**Review by thatlittleboy — APPROVED** [review]
thanks, lgtm. just a minor nitpick, but otherwise pre-approving

joey-friends-nice

File: janitor/functions/select.py (hunk 1 above)
Comment (samukweku) [reply janitor/functions/select.py:203]: @thatlittleboy which line are you referring to?

Comment (thatlittleboy) [reply janitor/functions/select.py:203]: @samukweku L203 (there are currently 2 new lines between the "Examples" section and the "Args" section, we just need one. :)

**Review by ericmjl — APPROVED** [review]

**Pull request merged**